src/runtime/env.rs

```diff
@@ -388,7 +388,7 @@ fn migrate_storage_schema_to_v6<E: Env>() -> TryEnvFuture<()> {
                 Some((Some(play_in_external_player), settings)) => {
                     settings.insert(
                         "playerType".to_owned(),
-                        if play_in_external_player == serde_json::Value::Bool(true) {
```

Comment:
@TheBeastLT you can also compare `serde_json::Value` directly to `bool`
https://docs.rs/serde_json/latest/serde_json/enum.Value.html#impl-PartialEq%3Cbool%3E-for-Value
I already had this change made (fixing the bug) that's why I left it in this PR

Reply:
Ahh nice, thanks! My IDE didn't resolve the type, so I didn't explore too much what is available.